packages/tabs/src/Tabs.stories.tsx

```diff
@@ -129,14 +129,28 @@ export const LiveExample: StoryFn<TabsProps> = ({
 );
 
 export const Controlled: StoryFn<TabsProps> = (args: TabsProps) => {
-  const [selectedTab, setSelectedTab] = useState(0);
+  const [selectedTab, setSelectedTab] = useState<string | number>(
```

Comment:
Can a consumer switch between using strings & numbers to reference tabs?

Reply:
Yes, they can.